harness/determined/profiler.py

```diff
@@ -48,6 +48,18 @@ class ShutdownMessage:
     pass
 
 
+def profiling_metrics_exist(master_url: str, trial_id: str) -> bool:
+    """
+    Return True if there are already profiling metrics for the trial.
+    """
+    # TODO: Implement
+    raise NotImplementedError
```

Comment:
Not sure is this was the plan after that PR yesterday, this could just call `GET:master_url/api/v1/trial/:trial_id/profiler/available_series?follow` and exit on any return, right?

Reply:
Updated the code